### PR TITLE
Updated the menu entry Firefox 87.0 compatible

### LINKED ISSUE #3474: Issue with "Firefox Developer Tools": possibly outdated information on menu item name
MDN URL: https://developer.mozilla.org/en-US/docs/Tools

#### What information was incorrect, unhelpful, or incomplete?
There's a phrase in the page: `You can open the Firefox Developer Tools from the menu by selecting Tools > Web Developer > Toggle Tools`. The last menu item's name in Firefox 87.0 is `Web Developer Tools`, not `Toggle Tools`.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/tools`
* MDN URL: https://developer.mozilla.org/en-US/docs/Tools
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/tools/index.html
* Last commit: https://github.com/mdn/content/commit/d354eede5b9fe8095e514664d307a1e6d2a32832
* Document last modified: 2021-02-02T17:10:43.000Z

</details>

### PR DESCRIPTION
**Pull request by CreaTorAlexander**
Fixes #3474

### CODE REVIEW

File: files/en-us/tools/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 <h2 id="The_Core_Tools">The Core Tools</h2>
 
-<p>You can open the Firefox Developer Tools from the menu by selecting <em>Tools</em> &gt; <em>Web Developer</em> &gt; <em>Toggle Tools</em> or use the keyboard shortcut  <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>I</kbd> or <kbd>F12</kbd> on Windows and Linux, or <kbd>Cmd</kbd> + <kbd>Opt</kbd> + <kbd>I</kbd> on macOS.</p>
+<p>You can open the Firefox Developer Tools from the menu by selecting <em>Tools</em> &gt; <em>Web Developer</em> &gt; <em>Web Developer Tools</em> or use the keyboard shortcut  <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>I</kbd> or <kbd>F12</kbd> on Windows and Linux, or <kbd>Cmd</kbd> + <kbd>Opt</kbd> + <kbd>I</kbd> on macOS.</p>
```
Comment (chrisdavidmills) [inline files/en-us/tools/index.html:27]: On my mac at least, the second menu level is labeled "Browser Tools", not "Web Developer". What does it say on yours, @CreaTorAlexander ?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Thanks @CreaTorAlexander ; small nitpick to consider here.

File: files/en-us/tools/index.html (hunk 1 above)
Comment (CreaTorAlexander) [reply files/en-us/tools/index.html:27]: I am currently on a linux machine and I can confirm that it has changed from *Toggle Tools* to *Web Developer Tools* there.
Later on I will check it on my mac. :+1:

Comment (CreaTorAlexander) [reply files/en-us/tools/index.html:27]: On my mac it is also *Web Developer Tools in Version 87*
Bildschirmfoto 2021-03-25 um 21 25 52

Comment (hamishwillee) [reply files/en-us/tools/index.html:27]: This is correct from FF87, which is now merged. Thanks @CreaTorAlexander - I recently also updated https://developer.mozilla.org/en-US/docs/Tools/Tools_Toolbox#toolbox with this info.

**Review by hamishwillee — APPROVED** [review]

**Pull request merged**